### PR TITLE
circleci: add centos6 as a new platform fo building ctags executable

### PR DESCRIPTION
**Pull request by masatake**
Signed-off-by: Masatake YAMATO <yamato@redhat.com>

### CODE REVIEW

**Comment by masatake** [comment]
This can be merged after merging #2798.

File: circle.yml
Comment (leleliu008) [inline circle.yml:242]: `autconf ` spelling mistake. Actually, `autoconf` will installed along with `automake`

Comment (masatake) [reply circle.yml:242]: Thank you!

File: circle.yml
Comment (leleliu008) [inline circle.yml:257]: Red Hat has added `Python 3.4` for `CentOS 6` through the `EPEL` repository.

easiest way to install `Python 3.4` on `CentOS 6` from `EPEL` repository:

```
yum install -y epel-release
yum install -y python34
```

File: circle.yml
Comment (leleliu008) [inline circle.yml:240]: `buildig ` spelling mistake

File: circle.yml
Comment (leleliu008) [inline circle.yml:238]: `centos:6` and `centos:6.10` are actually the same image.

here we just change `mirror` to `vault` is ok:

```
sed -i 's|mirror|vault|g' /etc/yum.repos.d/CentOS-Base.repo
```

File: circle.yml
Comment (masatake) [reply circle.yml]: Thank you.

Comment (masatake) [reply circle.yml]: I found our test harness needs python 3.5.
misc/units.py uses subprocess.run(), which is introduced in python3.5.
So I will not run the test cases on centos6@circleci.

File: circle.yml
Comment (masatake) [reply circle.yml:240]: Fixed. Thank you.

File: circle.yml
Comment (masatake) [reply circle.yml:238]: Fixed. Thank you.

**Review by leleliu008 — APPROVED** [review]

**Comment by masatake** [comment]
@leleliu008, thank you for your suggestions and review.
Many troubles about the building are gone. The last one is in our queue is only #2791.

**Comment by leleliu008** [comment]
nice!

**Pull request merged**
